utils/paramfetch/Cargo.toml

```diff
@@ -11,10 +11,12 @@ ahash.workspace = true
 anyhow.workspace = true
 backoff = { workspace = true, features = ["tokio"] }
 blake2b_simd.workspace = true
+cid = "0.10.1"
```

Comment:
Better to re-export `cid` from `rs-car` in case there is a version mismatch

Reply:
Done